docker/supervisor-distrepos.conf

```diff
@@ -1,5 +1,5 @@
 [program:distrepos]
-command=/bin/bash -c "/bin/distrepos --action rsync; /bin/distrepos --action mirror; sleep 360"
+command=/bin/bash -c "/bin/distrepos --action rsync; /bin/distrepos --action cadist; /bin/distrepos --action mirror; sleep 360"
```

Comment:
The initial multi-task behavior I implemented for `distrepos` was to bail early if any subcomponent failed. This might not be correct given that the actions are largely independent of each other, and the output of all 3 is suppressed anyways by the `sleep 360` at the end always returning 0.

Reply:
I think this is fine.  You should run the mirror action first though -- repo-update-cadist uses the repo definition files which refer to the mirror list, so if the mirror list files don't exist then repo-update-cadist will fail.